manual/cops_style.md

```diff
@@ -6028,6 +6028,28 @@ WordRegex | (?-mix:\A[\p{Word}\n\t]+\z)
 
 * [https://github.com/bbatsov/ruby-style-guide#percent-w](https://github.com/bbatsov/ruby-style-guide#percent-w)
 
+## Style/YodaCondition
+
+Enabled by default | Supports autocorrection
+--- | ---
+Enabled | No
```

Comment:
@betesh I've never done this before as well, so I don't know yet. But that would be good to add in later iterations.